src/openzaak/components/besluiten/tests/test_auth_cmis.py

```diff
@@ -30,6 +32,15 @@
 @override_settings(CMIS_ENABLED=True)
 class BesluitScopeForbiddenCMISTests(AuthCheckMixin, APICMISTestCase):
     def test_cannot_read_without_correct_scope(self):
+        Service.objects.create(
+            api_root="http://testserver/documenten/api/v1/", api_type=APITypes.drc
+        )
+        Service.objects.create(
+            api_root="http://testserver/catalogi/api/v1/", api_type=APITypes.ztc
+        )
+        Service.objects.create(
+            api_root="http://testserver/besluiten/api/v1/", api_type=APITypes.brc
+        )
```

Comment:
should we maybe make this part of `APICMISTestCase`? Just set up services for all API types and be done with it? :P